aea/configurations/schemas/aea-config_schema.json

```diff
@@ -151,6 +151,9 @@
     },
     "data_dir": {
       "type": "string"
+    },
+    "dependencies": {
+      "$ref": "definitions.json#/definitions/dependencies"
```

Comment:
this is currently optional for backward compatibility, but will be made mandatory in the next release.

Reply:
reason: otherwise many tests that rely on the remote registry would fail (e.g. `aea upgrade` tests).

Reply:
Yes, please open issue so we don't forget to make it strict later.